### PR TITLE
docs: add link for control flow migration

### LINKED ISSUE #53147: [Angular.dev] Missing page for control-flow migration
### Describe the problem that you experienced

There should be a page on https://angular.dev/reference/migrations -> https://angular.dev/reference/migrations/control-flow mentioning the `ng generate @angular/core:control-flow` migration mentioned in the [blog post](https://blog.angular.io/introducing-angular-v17-4d7033312e4b).

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: fixes #53147

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/src/content/reference/migrations/control-fow.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Migration to Control Flow syntax
+
+
+[Control flow syntax](../../guide/templates/control-flow) is available from Angular 17 and simplified way to use the control-flow directives like *ngFor, *ngIf and *ngSwitch.
```
Comment (JeanMeche) [inline adev/src/content/reference/migrations/control-fow.md:4]: I believe the link should be relative to the root. 
```suggestion
[Control flow syntax](guide/templates/control-flow) is available from Angular 17 and simplified way to use the control-flow directives like *ngFor, *ngIf and *ngSwitch. 
```

Comment (jessicajaniuk) [reply adev/src/content/reference/migrations/control-fow.md:4]: Agree. if you can apply these suggestions, I'll approve.

Comment (jessicajaniuk) [reply adev/src/content/reference/migrations/control-fow.md]: Yes, please squash the commits.

Comment (santoshyadavdev) [reply adev/src/content/reference/migrations/control-fow.md]: Done

**Review by jessicajaniuk — APPROVED** [review]
LGTM! Thank you!

**Comment by jessicajaniuk** [comment]
@santoshyadavdev  Looks like the link is actually incorrect in the preview. Can you fix?

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
The preview shows the link to the control flow landing page to be broken. This needs to be fixed before we can merge.

**Comment by santoshyadavdev** [comment]
@jessicajaniuk sure I will get it fixed in morning

**Comment by santoshyadavdev** [comment]
> @jessicajaniuk sure I will get it fixed in morning

sorry @jessicajaniuk I ended up screwing up this branch, and ported changes to this PR https://github.com/angular/angular/pull/54108

**Review by jessicajaniuk — APPROVED** [review]
Thanks for taking care of this! Looks like the link works now.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit d9a84705c969a8fdc6dfa37a85fc20e903cdbc04.

**Pull request closed without merging**